crates/flox/src/commands/environment.rs

```diff
@@ -203,9 +203,8 @@ impl EnvironmentCommands {
                     .catalog(&flox.nix(Default::default()), &flox.system)
                     .await
                     .context("Could not get catalog")?;
-                // let installed_store_paths = env.installed_store_paths(&flox).await?;
 
-                println!("Packages in {}:", env.environment_ref());
+                println!("Packages in {}:", env.name());
                 for (publish_element, _) in catalog.entries.iter() {
                     if publish_element.version != LATEST_VERSION {
```

Comment:
I think we could add a few more things present in bash list:
- Add path
- Formatting of columns when versions are present
- Maybe system - I'm actually kind of inclined to not add that but it's present in existing output

We could always come back and do those things later, but I think they'll likely be non-regrettable

Reply:
Sure, I was just going off the acceptance criteria in the ticket

Reply:
Yeah I'm fine either way

Reply:
> Add path

Which path are you referring to here?

Reply:
It already displays as:
```
Packages for <env>:
<stability>.<package name>
...
```

Reply:
> > Add path
> 
> Which path are you referring to here?

The path of the environment. /some/path/.flox/name

Reply:
I'm not sure how much I like this `Packages for ...` line anyway. You can imagine someone piping the output of `flox list` into some line-based text processing pipeline line `flox list | tr -s ' ' | cut -d ' ' -f 1` just to get the package names, and having this first line messes that up.

Also, with the current feature-flagged commands there are no versions and the stabilities are all `stable`... I'm not sure if that's supposed to be how things are working right now.

Reply:
Yeah I think we'd either have exclusively a list of packages output or something more similar to existing output. Although I also think for piping we can potentially rely on json

Reply:
It looks like with recent changes we no longer have stabilities, all `install`s are coming from `flox-nixpkgs`, so there's not really much to display here. I'd just punt on this and wait until there's actually something to display.